tests/components/onewire/test_select.py

```diff
@@ -42,6 +44,31 @@ async def test_selects(
     await snapshot_platform(hass, entity_registry, snapshot, config_entry.entry_id)
 
 
+@pytest.mark.parametrize("device_id", ["28.111111111111"])
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_selects_delayed(
+    hass: HomeAssistant,
+    config_entry: MockConfigEntry,
+    owproxy: MagicMock,
+    device_id: str,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test for delayed 1-Wire select entities."""
+    setup_owproxy_mock_devices(owproxy, [])
+    await hass.config_entries.async_setup(config_entry.entry_id)
+
+    assert not er.async_entries_for_config_entry(entity_registry, config_entry.entry_id)
+
+    setup_owproxy_mock_devices(owproxy, [device_id])
+    freezer.tick(_DEVICE_SCAN_INTERVAL)
+    async_fire_time_changed(hass)
+    await hass.async_block_till_done(wait_background_tasks=True)
+
+    await snapshot_platform(hass, entity_registry, snapshot, config_entry.entry_id)
```

Comment:
Let's not add more snapshots unless there is any relevance in storing the data. Because I think this test would also be good if we just checked that we discovered them. This avoid unneeded snapshots

Reply:
> Let's not add more snapshots unless there is any relevance in storing the data. Because I think this test would also be good if we just checked that we discovered them. This avoid unneeded snapshots

Removed